test/betterbench_avx.cpp

```
@@ -0,0 +1,60 @@
+    };
+
+    std::size_t i = 0;
+    {
+        MKN_KUL_DBG_FUNC_ENTER;
+
+        for (; i < arrs; ++i)
+            op(d0[i](), d1[i](), d2[i](), d3[i]());
+    }
+
+    if (d3[arrs - 1][3] != b3)
+        throw std::runtime_error("FAIL!");
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)